RELEASES.md

```diff
@@ -1,3 +1,166 @@
+Version 1.0.0-alpha (January 2015)
+----------------------------------
+
+  * ~2300 changes, numerous bugfixes
+
+  * Highlights
+
+    * The language itself is considered feature complete for 1.0,
+      though there is a significant amount of cleanup and bugfixes
+      remaining.
+    * Nearly 50% of the public API surface of the standard library has
+      been declared 'stable'. Those interfaces will not change.
+    * Most crates that are not `std` have been moved out of the Rust
+      distribution into the Cargo ecosystem so they can evolve
+      separately and don't need to be stabilized as quickly, including
+      'time', 'getopts', 'num', 'regex', and 'term'.
+    * Documentation continues to be expanded with more guides, more
+      API coverage and more examples.
+    * All official Rust binary installers now come with [Cargo], the
+      Rust package manager.
+
+* Language
+
+    * Closures have been [completely redesigned][unboxed] to be
+      implemented in terms of traits, can now be used as generic type
+      bounds and thus monomorphized and inlined, or via an opaque
+      pointer (boxed) as in the old system. The new system is often
+      referred to as 'unboxed' closures.
+    * Enum variants are [namespaced by their type names][enum].
+    * [`where` clauses][where] provide a more versatile and attractive
+      syntax for specifying generic bounds, though the previous syntax
+      remains valid.
+    * Rust again picks a [fallback] (either i32 or f64) for uninferred
```

Comment:
Should be `[fallback][fallback]` so the `(either i32 or f64)` part isn't parsed as the link's URL.